mne/defaults.py

```diff
@@ -60,6 +62,8 @@
         fnirs_fd_ac_amplitude="V",
         fnirs_fd_phase="rad",
         fnirs_od="V",
+        fnirs_td_gated_amplitude="M",
+        fnirs_td_moments_amplitude="M",
```

Comment:
this is a bit weird, the dataType `Time domain – Moments (TD Moments) - Amplitude` can be any of the moments, which all have different units:
- sum or intensity is photon counts
- mean is s
- variance is s^2

Reply:
Ahhh... in this case we probably need to update the FIF "coil type" constants first since these are really sensing/detecting different sorts of data. Like we'll probably need:

- `td_moments_intensity` (photon counts)
- `td_moments_mean` (s)
- `td_moments_variance` (s^2)

or similar